backend/signature/signature-lambda/build.gradle.kts

```diff
@@ -4,9 +4,14 @@ plugins {
     id("io.quarkus")
 }
 
+repositories {
```

Comment:
It's already defined in eln-conventions.gradle,kts, that should be applied automatically, so looks like not needed

Reply:
Please remove from signature-core also